packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [45.1.0]
+
+### Added
+
+- Add support for EIP-7702 / type 4 transactions ([#5285](https://github.com/MetaMask/core/pull/5285))
+  - Add `setCode` to `TransactionEnvelopeType`.
+  - Add `authorizationList` to `TransactionParams`.
+
+### Changed
+
+- Bump `@metamask/base-controller` from `^7.1.1` to `^8.0.0` ([#5305](https://github.com/MetaMask/core/pull/5305))
```

Comment:
It looks like 5285 contained [breaking changes to the messenger](https://github.com/MetaMask/core/pull/5285/files#diff-96bc10c64242a8e23f5835532988319189239bdf40f556b86199dcbcc41c7bfcL340) that aren't captured here:

``` suggestion
- **BREAKING:** The TransactionController messenger must now allow the `KeyringController:signAuthorization` action ([#5285](https://github.com/MetaMask/core/pull/5285))
- Bump `@metamask/base-controller` from `^7.1.1` to `^8.0.0` ([#5305](https://github.com/MetaMask/core/pull/5305))
```

@matthewwalsh0 Can you confirm this?

Reply:
fixed

Reply:
I believe these are breaking as that flow isn't yet utilised in any client meaning that permission is never required.

Reply:
@matthewwalsh0 breaking change rolled back here https://github.com/MetaMask/core/pull/5305/commits/7f59b09569f3eb48ac896ebfef98998471913bce